python-checks/src/main/java/org/sonar/python/checks/IdenticalExpressionOnBinaryOperatorCheck.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * SonarQube Python Plugin
+ * Copyright (C) 2011-2016 SonarSource SA
+ * mailto:contact AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.python.checks;
+
+import com.google.common.collect.ImmutableList;
+import com.sonar.sslr.api.AstNode;
+import org.sonar.check.Priority;
+import org.sonar.check.Rule;
+import org.sonar.python.PythonCheck;
+import org.sonar.python.api.PythonGrammar;
+import org.sonar.squidbridge.annotations.ActivatedByDefault;
+import org.sonar.squidbridge.annotations.SqaleConstantRemediation;
+
+import java.util.List;
+
+@Rule(
+  key = "S1764",
+  name = "Identical expressions should not be used on both sides of a binary operator",
+  priority = Priority.MAJOR,
+  tags = {Tags.BUG, Tags.CERT})
+@SqaleConstantRemediation("2min")
+@ActivatedByDefault
+public class IdenticalExpressionOnBinaryOperatorCheck extends PythonCheck {
+
+  private static final List<String> EXCLUDED_OPERATOR_TYPES = ImmutableList.of(
+    "*",
+    "+"
+  );
+
+  @Override
+  public void init() {
+    subscribeTo(
+      PythonGrammar.M_EXPR,
+      PythonGrammar.A_EXPR,
+      PythonGrammar.SHIFT_EXPR,
+      PythonGrammar.AND_EXPR,
```

Comment:
@ivandalbosco I think we should raise an issue, it could be a copy paste mistake.